.travis.yml

```diff
@@ -47,14 +47,13 @@ before_install:
       sudo pip install cpp-coveralls requests[security];
     else
       brew update; brew update;
-      brew install glew wxwidgets llvm36;
+      brew install glew wxwidgets;
     fi;
 
 before_script:
- - git submodule update --init asmjit ffmpeg
+ - git submodule update --init asmjit ffmpeg llvm
  - mkdir build
  - cd build
- - if [ "$TRAVIS_OS_NAME" = "linux" ]; then cmake ..; else cmake .. -DLLVM_DIR=/usr/local/opt/llvm36/lib/llvm-3.6/share/llvm/cmake; fi
```

Comment:
@Zangetsu38 Do you even understand how CMake and such work? Don't try to "fix" stuff, that is needed or is not a problem or broken.